src/Raven.Client/Data/IndexQuery.cs

```diff
@@ -261,14 +256,12 @@ public void AppendQueryString(StringBuilder path, bool includePageSizeEvenIfNotE
             {
                 foreach (var field in DynamicMapReduceFields)
                 {
-                    if (field.IsGroupBy)
-                        path.Append("&groupBy=").Append(Uri.EscapeDataString(field.Name));
-                    else
-                    {
-                        path.Append("&fetch=").Append(Uri.EscapeDataString(field.Name));
-                        path.Append("/");
-                        path.Append(field.OperationType);
-                    }
+                    path.Append("&mapReduce=")
```

Comment:
Because I had an inconsistency between how I kept this data on the client side (`IndexQuery.DynamicMapReduceFields` - a dedicated structure for dynamic map-reduce fields) and server side (`IndexQuery.FieldsToFetch` with "convention" that a field's name and its operation are separated by `/` character and `IndexQuery.GroupByFields` used only on the server side for that purpose). I found it quite counterintuitive.  After this change, we immediately know that the query is a dynamic map-reduce one and can parse things into the same structure - `DynamicMapReduceField[]`.